docs/setup/navqplus.md

```diff
@@ -1,5 +1,8 @@
 # Connect Create® 3 to NavQ+ and set up ROS 2 Galactic
 
+!!! important
+This is community-submitted content. Please feel welcome to submit PRs for additions or corrections.
```

Comment:
sorry -- did not notice that GH interpreted my markdown as, well, markdown in the comment. The syntax should be

```
!!! important
    **This is community-submitted content. Please feel welcome to submit PRs for additions or corrections.**
```

See https://github.com/iRobotEducation/create3_docs/edit/main/docs/setup/jetson.md for further clarification

Reply:
Actually that's all on me, I was lazy and should have previewed it first.

Fixed here:
[3127c4b](https://github.com/iRobotEducation/create3_docs/pull/166/commits/3127c4b79939c2b61f6b4993224f51dba61e50b3)